alacritty_config_derive/src/serde_replace.rs

```diff
@@ -98,9 +99,28 @@ fn match_arms<T>(fields: &Punctuated<Field, T>) -> TokenStream2 {
                 _ => alacritty_config::SerdeReplace::replace(&mut self.#ident, value)?,
             });
         } else {
-            stream.extend(quote! {
-                #literal => alacritty_config::SerdeReplace::replace(&mut self.#ident, next_value)?,
-            });
+            // Extract all `#[config(alias = "...")]` attribute values.
+            let aliases = field
+                .attrs
+                .iter()
+                .filter(|attr| (*attr).path().is_ident("config"))
+                .filter_map(|attr| attr.parse_args::<Attr>().ok())
+                .filter(|parsed| parsed.ident.as_str() == "alias")
+                .map(|parsed| parsed.param.map(|v| v.value()))
+                .collect::<Option<Vec<_>>>();
```

Comment:
This will be `None` if *any* of the attributes has no value. That seems incorrect to me.

It should just be collected into a vec and empty elements should be skipped. Or even better a compile error is produced if there is no value.

Reply:
@chrisduerr, so when encounter `#[config(alias = "")]` we should panic?